scripts/sync.py

```diff
@@ -123,17 +123,14 @@ def main():
     # run downloader once, or continuously
     if args.continuous:
         while True:
-            do(args)
+            do(args, Lochness)
             logger.info('sleeping for {0} seconds'.format(Lochness['poll_interval']))
             time.sleep(Lochness['poll_interval'])
     else:
         do(args)
```

Comment:
Musn't this be `do(args, Lochness)` as well? I mean even I do not use the `--continuous` flag.

Reply:
Thanks for spotting this. Fixed at https://github.com/PREDICT-DPACC/lochness/pull/37/commits/79332d271496685994005e87f0fc9382a3aefbf4